v22.1/deploy-app-vercel.md

```diff
@@ -0,0 +1,177 @@
+{{site.data.alerts.callout_info}}
+The connection string is pre-populated with your username, cluster name, and other details, including your password. Your password, in particular, will be provided only once. Save it in a secure place (we recommend a password manager) to connect to your cluster in the future. If you forget your password, you can reset it by going to the [**SQL Users** page](../cockroachcloud/user-authorization.html).
+{{site.data.alerts.end}}
+
+## Step 2. Get the code
+
+1. Clone the `cockroachdb` fork of the `prisma-examples` repo:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ git clone git@github.com:cockroachdb/prisma-examples.git
+    ~~~
+
+    {{site.data.alerts.callout_info}}
+    In this tutorial, we deploy a slightly modified version of Prisma's [`rest-nextjs-api-routes`](https://github.com/prisma/prisma-examples/tree/latest/typescript/rest-nextjs-api-routes) example app.
+    {{site.data.alerts.end}}
+
+1. Navigate to the `rest-nextjs-api-routes` directory:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ cd typescript/rest-nextjs-api-routes
+    ~~~
+
+## Step 3. Initialize the database
+
+1. Set the local `DATABASE_URL` environment variable to [the connection string](#connection-string) you obtained earlier from the {{ site.data.products.db }} Console:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ export DATABASE_URL="<connection-string>"
+    ~~~
+
+    By default, Prisma uses the local `DATABASE_URL` environment variable as the connection string to the database.
+
+1. Install Prisma:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    npm install prisma --save-dev
+    ~~~
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    npx prisma
+    ~~~
+
+1. Run [Prisma Migrate](https://www.prisma.io/docs/concepts/components/prisma-migrate) to initialize the database with the schema defined in `prisma/prisma.schema`.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ npx prisma migrate dev --name init
+    ~~~
+
+    You should see the following output:
+
+    ~~~
+    Your database is now in sync with your schema.
+
+    ✔ Generated Prisma Client (3.11.0 | library) to ./node_modules/@prisma/client in 87ms
+    ~~~
+
+    This command also initializes [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) to communicate with your CockroachDB cluster, based on the configuration in the `prisma/schema.prisma` file.
+
+## Step 4. Deploy the application
+
+1. Save the local `DATABASE_URL` environment variable to a `.env` file in your project:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ echo "DATABASE_URL=$DATABASE_URL" >> .env
+    ~~~
+
+    Next.js loads the variables defined in `.env` to the project environment.
+
+1. (Optional) Run the app server locally to preview your site:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ npm run dev
```

Comment:
One thing that might be nice would be how to tell when the server is up. In this case I think they'll see a line like, `ready - started server on 0.0.0.0:3000, url: http://localhost:3000`.